src/algokit/cli/init.py

```diff
@@ -85,10 +108,11 @@ def _get_blessed_templates() -> dict[str, BlessedTemplateSource]:
 
 
 def validate_dir_name(context: click.Context, param: click.Parameter, value: str | None) -> str | None:
-    if value is not None and not re.match(r"^[\w\-.]+$", value):
+    if value is not None and not re.match(r"^[\w\-.\\/]+(?:[\w\-. ]+)*$", value):
```

Comment:
these are to also allow specifying a target folder as a path - currently it always assumes that if its a folder in local fs it has to be relative to root. This change allows setting this param as 'path/to/some/nested/dir' - this way can also ensure that when fullstack is doing init on child templates instead of doing `mkdir` checks to drop them into a workspace structure it can specify the nested path directly in the name parameter

Reply:
this is now reverted back to original implementation, a different method takes care of moving the project to expected projects folder inside the workspace without needed to modify the existing init command semantics